pubspec.lock

```diff
@@ -191,12 +149,5 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "2.0.8"
-  xml:
-    dependency: transitive
-    description:
-      name: xml
-      url: "https://pub.dartlang.org"
-    source: hosted
-    version: "3.5.0"
 sdks:
-  dart: ">=2.4.0 <3.0.0"
+  dart: ">=2.2.2 <3.0.0"
```

Comment:
Sorry I'm just now getting to reviewing this. Was there a package that required us to bump min Dart version down to 2.2.2? We lose some helpful syntactic sugar under 2.3.0... Perhaps set 
```
environment:
  sdk: ">=2.4.0 <3.0.0"
```
in `pubspec.yaml`

Reply:
Thanks for noticing that! I did not intend to make this change. It may have happened when I tried to build the project.